controllers/v1/observationsController.js

```diff
@@ -1035,7 +1036,7 @@ module.exports = class Observations extends Abstract {
           }
         }
 
-        const parsedAssessment = await assessmentsHelper.parseQuestions(
+        const parsedAssessment = await assessmentsHelper.parseQuestionsV2(
```

Comment:
@borkarsaish65 why controller v1 is calling helper v2?

Reply:
This change is done by @praveenKDass . It was done as page question were not appearing.

Reply:
@VISHNUDAS-tunerlabs  This changes has been implemented already in sunbird, if we only calls parseQuestions it will give only questions in response we have modified that in parseQuestionsV2 to get as a page questions